src/js/components/Select/SelectContainer.js

```diff
@@ -131,31 +153,35 @@ class SelectContainer extends Component {
       >
         <Box
           id={id ? `${id}__select-drop` : undefined}
-          background={dropBackground}
         >
-          {onSearch ? (
-            <Box pad='xsmall'>
-              <TextInput
-                focusIndicator={true}
+          {onSearch && (
+            <Box pad={!customSearchInput ? 'xsmall' : undefined} flex={false}>
+              <SelectTextInput
+                focusIndicator={!customSearchInput}
                 size='small'
-                ref={(ref) => { this.searchRef = ref; }}
+                ref={this.searchRef}
                 type='search'
                 value={search}
                 placeholder={searchPlaceholder}
                 onInput={this.onInput}
               />
             </Box>
-          ) : undefined}
-
+          )}
           <Box
-            flex={false}
+            flex={true}
```

Comment:
This seems like a notable change. Why was this changed?

Reply:
I'm not sure why we added flex false before. but I'm certain that it was to avoid elements collapsing.
But it makes sense for the search input to be fixed and the list of options to scroll. the list of options should take the remaining space in the drop height.